extensions-contrib/graphite-emitter/src/main/java/io/druid/emitter/graphite/GraphiteEmitter.java

```diff
@@ -175,11 +173,12 @@ public void run()
             if (e instanceof InterruptedException) {
               Thread.currentThread().interrupt();
```

Comment:
What practically happens after this statement? We continue with the next iteration in the loop?